src/forms/FormNav/FormNav.tsx

```diff
@@ -88,6 +90,7 @@ const FormNav = (props: FormNavProps): ReactElement => {
 				const sectionHeight = currentSection?.offsetHeight;
 				const sectionTop = currentSection?.offsetTop - 350;
 				const sectionId = currentSection?.getAttribute("id");
+

```

Comment:
Found this other approach using observers, we might need to rethink / refactor our use effect. It's compatible with all browsers and I think might be a more "elegant" solution than the one I suggested above, but we'll have to give them a try.